specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json

```diff
@@ -0,0 +1,1940 @@
+          "minimum": 1,
+          "x-nullable": false
+        },
+        "columns": {
+          "description": "Number of columns.",
+          "type": "integer",
+          "minimum": 1,
+          "x-nullable": false
+        },
+        "cells": {
+          "description": "List of cells contained in the table.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/DataTableCell"
+          }
+        }
+      }
+    },
+    "DataTableCell": {
+      "description": "Information about the extracted cell in a table.",
+      "type": "object",
+      "required": [
+        "rowIndex",
+        "columnIndex",
+        "text",
+        "boundingBox",
+        "confidence"
+      ],
+      "properties": {
+        "rowIndex": {
+          "description": "Row index of the cell.",
+          "type": "integer",
+          "minimum": 0,
+          "x-nullable": false
+        },
+        "columnIndex": {
+          "description": "Column index of the cell.",
+          "type": "integer",
+          "minimum": 0,
+          "x-nullable": false
+        },
+        "rowSpan": {
+          "description": "Number of rows spanned by this cell.",
+          "type": "integer",
+          "minimum": 1,
+          "default": 1,
+          "x-nullable": false
+        },
+        "columnSpan": {
+          "description": "Number of columns spanned by this cell.",
+          "type": "integer",
+          "minimum": 1,
+          "default": 1,
+          "x-nullable": false
+        },
+        "text": {
+          "description": "Text content of the cell.",
+          "type": "string"
+        },
+        "boundingBox": {
+          "description": "Bounding box of the cell.",
+          "$ref": "#/definitions/BoundingBox"
+        },
+        "confidence": {
+          "description": "Confidence value.",
+          "$ref": "#/definitions/Confidence"
+        },
+        "elements": {
+          "description": "When includeTextDetails is set to true, a list of references to the text elements constituting this table cell.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ElementReference"
+          }
+        },
+        "isHeader": {
+          "description": "Is the current cell a header cell?",
+          "type": "boolean",
+          "default": false,
+          "x-nullable": false
+        },
+        "isFooter": {
+          "description": "Is the current cell a footer cell?",
+          "type": "boolean",
+          "default": false,
+          "x-nullable": false
+        }
+      }
+    },
+    "DocumentResult": {
+      "description": "A set of extracted fields corresponding to the input document.",
+      "type": "object",
+      "required": [
+        "docType",
+        "pageRange",
+        "fields"
+      ],
+      "properties": {
+        "docType": {
+          "description": "Document type.",
+          "type": "string"
+        },
+        "modelId": {
```

Comment:
If I wanted to know if my model is composed or not (not sure if this is a real customer need), do I make a get request to get the ModelInfo for the modelId?  Is that information available in ModelInfo, or do customers need to track this themselves?

Reply:
You can determine this by checking if Model contains trainResult or composedTrainedResults.  Had we planned ahead, a better design is to check the number of objects under a merged trainResults.  Unfortunately, we cannot break the existing interface.